Framework/CurveFitting/test/Algorithms/PlotPeakByLogValueTest.h

```diff
@@ -615,6 +590,30 @@ class PlotPeakByLogValueTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().clear();
   }
 
+  void test_exclude_range() {
```

Comment:
A propositional test name i.e. `test_excluding_range_does_some_thing` makes it clearer what the test is testing. It appears this one just simply checks that you can enter an excluded range and the algorithm still runs. Could there be more tests on the actual results of the fit? Also, are there any fail cases that need to be checked like giving an invalid range?